tests/signals_connect/screen-output

```diff
@@ -0,0 +1,46 @@
+-----------------------------------------------------------------------------
+-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
+--     . version 1.4.pre
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+--     . using Trilinos
+-----------------------------------------------------------------------------
+
+Loading shared library <./libsignals_connect.so>
+
+Connecting signals
+Number of active cells: 16 (on 3 levels)
+Number of degrees of freedom: 268 (162+25+81)
+
+
+*** constraints signal called ***
+
+*** Timestep 0:  t=0 seconds
+
+*** constraints signal called ***
```

Comment:
why is this triggered twice? Is this ```constraints``` vs. ```current_constraints```? Should we do one more time step to see it is triggered again?